drivers/src/ecc384.rs

```diff
@@ -376,9 +378,42 @@ impl Ecc384 {
         digest: &Ecc384Scalar,
         signature: &Ecc384Signature,
     ) -> CaliptraResult<Ecc384Result> {
+        // Get the verify r result
```

Comment:
Can we #[inline(always)] this function?

Reply:
Hmm, maybe I'm misunderstand how you're planning on using verify_r

I originally thought verify() was going to be inlined, but I guess instead you're planning to call verify_r directly in a future PR?